### ISSUE TITLE
VSCode Server Fails to Start When Creating Docker Project: "/vscode/1.89.1.24130/node: not found"

### ISSUE THREAD

**Issue by thefatdeveloper** [codesandbox/codesandbox-client#8616]
![image](https://github.com/user-attachments/assets/a2bdee67-8b75-4799-ba15-54168d323056)

# 🐛 bug report
## Preflight Checklist
- [x] I have read the [Contributing Guidelines][contributing] for this project.
- [x] I agree to follow the [Code of Conduct][code_of_conduct] that this project adheres to.
- [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

## Description of the problem
When attempting to create a project in CodeSandbox using the Docker template, the process gets stuck at the VS Code server boot up stage. The server fails to start, preventing the project from being created.

## How has this issue affected me ? What am I trying to accomplish?
This issue is preventing me from creating and working on a Docker-based project in CodeSandbox. I'm unable to proceed with my development work due to this error.

### To Reproduce
1. Open CodeSandbox
2. Attempt to create a new project using the Docker template
3. Wait for the project to initialize
4. Observe that the process gets stuck at "Starting VSCode server..."
5. Check the console output, which shows an error: "/vscode/1.89.1.24130/node: not found"

### Link to sandbox: [link]() (optional)
(https://codesandbox.io/p/github/thefatdeveloper/nginx-docker-example/main?workspaceId=17d20f2e-9c30-4441-8067-800d0e9d3ceb)

Need Help in resolving this.

**Comment by ProchaLu** [comment]
When starting a project directly from GitHub and changing the URL to `githubbox`, I also encounter the following error:

```
> /vscode/1.89.1.24130/bin/codium-server --port 50000-50100 --connection-token-file /var/run/pitcher/.vscode/1.89.1.24130/.token
/vscode/1.89.1.24130/bin/codium-server: line 12: /vscode/1.89.1.24130/node: not found
```

As a workaround, you can skip it by clicking on "Skip VS Code initialization"

**Comment by thefatdeveloper** [comment]
Yeah, i tried skipping that step, but problem is GIT workflow will fail and you will not be able to checkin any code back :(

**Comment by filipelima18** [comment]
Hi @thefatdeveloper, Are you still experiencing this issue? I was unable to reproduce it. Clearing your browser cache and cookies might help - but let me know if that doesn't work and we can look into it.

**Comment by ProchaLu** [comment]
@filipelima18 Tried it with the incognito mode of Google Chrome, didn't experience any issue
